cdap-app-fabric/pom.xml

```diff
@@ -73,6 +73,16 @@
       <artifactId>cdap-notifications</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-kubernetes</artifactId>
```

Comment:
Don't add dependency on the kubernetes module. Everything should just go through the `cdap-master-spi` module.